src/assets/styles/theme.scss

```diff
@@ -9,6 +9,12 @@ $breakpoint-tablet: 700px;
 // scheme for color names
 // color-name_color-intensiveness_transparency
 
+$dark-navbar-mobile:rgba(8,0,16);
+
+$grey_25: #999999;
```

Comment:
jak nazwa tego koloru istnieje a istnieje niżej to układamy kolorki wg numeru od najmniejszego